server/schedulers/hot_region.go

```diff
@@ -329,10 +331,15 @@ func (h *hotScheduler) balanceHotWriteRegions(cluster schedule.Cluster) []*opera
 }
 
 type solution struct {
-	srcStore    *statistics.StoreLoadDetail
-	srcPeerStat *statistics.HotPeerStat
-	region      *core.RegionInfo
-	dstStore    *statistics.StoreLoadDetail
+	srcStore     *statistics.StoreLoadDetail
+	region       *core.RegionInfo // The region of the main balance effect. Relate mainPeerStat. srcStore -> dstStore
+	mainPeerStat *statistics.HotPeerStat
+
+	dstStore        *statistics.StoreLoadDetail
+	revertRegions   []*core.RegionInfo // The regions to hedge back effects. Relate revertPeersStat. dstStore -> srcStore
+	revertPeersStat []*statistics.HotPeerStat
```

Comment:
```suggestion
	revertPeersStat *statistics.HotPeerStat
```
I prefer it like this. if you want to support more operators, may also need to make `mainPeerStat` to `[]*statistics.HotPeerStat` in the future.